driver/pace/driver/initialization.py

```diff
@@ -77,10 +77,7 @@ class BaroclinicConfig(Initializer):
     Configuration for baroclinic initialization.
     """
 
-    @property
-    def start_time(self) -> datetime:
-        # TODO: instead of arbitrary start time, enable use of timedeltas
-        return datetime(2000, 1, 1)
+    start_time: datetime = datetime(2000, 1, 1)
```

Comment:
No change needed, but for my information can `datetime` be specified in yaml? I excluded it before because I thought it's either not possible or a pain to do.

Reply:
It worked if the format is correct, I have driver config writes out the restart yaml, and the format is this: `2000-01-01 00:07:30`. Then when it reads, it's able to parse that as `datetime`.

Reply:
Wow! Cool!